src/app/modules/admin/components/manage-session/manage-session.component.html

```diff
@@ -1,5 +1,5 @@
 <app-page-header [config]="headerConfig">
-  <div class="d-flex flex-justify-center my-8">
+  <div class="d-flex flex-justify-center my-8" *ngIf="segmentType === 'manage-session'">
```

Comment:
What about other places? Search is not working there too right? What was the conclusion on the call?